src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs

```diff
@@ -69,17 +69,20 @@ public async Task<Result<InvitationDto>> Handle(GetInvitationByIdQuery request,
             {
                 meeting = await _meetingClient.GetMeetingAsync(invitation.MeetingId,
                     query => query.ExpandInviteBodyHtml().ExpandProperty("participants.outlookstatus"));
-                LogFusionMeeting(meeting);
+                if (meeting != null)
```

Comment:
What happens if a meeting is null? Should it be logged, does it cause issues later in the method?

Reply:
There is no specific logging here when meeting is null. But there are other evidences being logged. meeting null does not cause issues later in the method.